botfront/imports/ui/components/nlu/models/Filters.jsx

```diff
@@ -30,12 +30,14 @@ export default class Filters extends React.Component {
     };
 
     render() {
-        const { filter: { intents: intentsFilter, entities: entitiesFilter, query }, intents, entities } = this.props;
+        const {
+            filter: { intents: intentsFilter, entities: entitiesFilter, query }, intents, entities, className,
+        } = this.props;
         const renderIntentLabel = label => ({ color: 'purple', content: `${label.text}` });
         const renderEntityLabel = label => ({ color: getColor(label.text, true), content: `${label.text}` });
 
         return (
-            <div style={{ marginRight: '10px' }}>
+            <div className={className}>
```

Comment:
can you add the classname directly here and just use the classname prop of Filters as an optional additional class ( use a template literal )

Reply:
because here we can't add a class with the prop, we can just overwrite the existing ones

Reply:
It's just that in my use case I have to use 'side-by-side right narrow' elsewhere, so I can't do e.g. 'side-by-side left narrow right'